config/internal/configsource/README.md

```diff
@@ -0,0 +1,101 @@
+| |    # source that insert YAML from env vars.     | |                |                                 |||||
+| |                                                 | |                |                                 |||||
+| |# Below the standard YAML configuration but      | |                |                                 |||||
+| |# some data still to be retrieved from the       | |                |                                 |||||
+| |# config sources.                                | |                |                                 |||||
+| |                                                 | |                |      configsource Factory       |||||
+| |receivers: ${include:/cfgs/rcvrs/def.yaml}       | |                |                                 |||||
+| |                                                 | |                |                                 |||||
+| |exporters ${env:EXPORTERS_DEFINITION}            | |                |                                 |||||
+| |                                                 | |                |                                 |||||
+| |service:                                         | |                |                                 |||||
+| |  pipelines:                                     | |                |                                 |||||
+| |    trace:                                       | |                |                                 ||||+
+| |      receivers: ${include:/cfgs/rcvrs/use.yaml} | |                |                                 |||+ 
+| |      exporters: ${env:EXPORTERS_IN_USE}         | |                |                                 ||+  
+| +-------------------------------------------------+ |                |                                 |+   
++-----------------------------------------------------+                +---------------------------------+    
+                       |                                                                |                     
+                       |                                                                |                     
+                       +----------------------------------------------------------------+                     
+                                                     |                                                        
+                                                     |                                    +------------------+
+                                                     v                                   +------------------+|
+                                     +-------------------------------+                  +------------------+||
+                                     |                               |                  |                  |||
+                                     |      configsource.Manager     |                  | configsource Obj ||+
```

Comment:
What's the relations between `configsource.Manager` and `configsource Obj` here? There is no arrow, os it is a bit unclear what happens here. Perhaps explain in the text?

Reply:
Trying to express that `configsource.Manager` creates the `configsource Obj` from the initial configuration. I will add some text and perhaps a separate diagram.

Reply:
I opted to remove it from this diagram and explain with text below.